sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -734,28 +726,30 @@ def _get_labels_inertia(
 
         # inertia = per_sample_inertia.sum()
         inertia = dpt.asnumpy(reduce_inertia_kernel(per_sample_inertia))
-        # inertia is now a 1-sized numpy array, we transform it into a scalar:
-        inertia = inertia.astype(output_dtype)[0]
 
         return assignments_idx, inertia
 
     def get_euclidean_distances(self, X, Y):
 
-        euclidean_distances, output_dtype = self._get_euclidean_distances(
+        (X, _, Y, output_dtype) = self._check_inputs(
             X,
-            Y,
+            sample_weight=_IgnoreSampleWeight,
+            cluster_centers=Y,
         )
 
-        return dpt.asnumpy(euclidean_distances).astype(output_dtype, copy=False)
+        X_t, _, Y_t = self._load_transposed_data_to_device(X, None, Y)
 
-    def _get_euclidean_distances(self, X, Y):
-        (X, _, Y, compute_dtype, output_dtype, work_group_size,) = self._check_inputs(
-            X,
-            sample_weight=_IgnoreSampleWeight,
-            cluster_centers=Y,
+        euclidean_distances = self._get_euclidean_distances(
+            X_t,
+            Y_t,
         )
```

Comment:
```suggestion
        euclidean_distances = self._get_euclidean_distances(X_t, Y_t)
```

Reply:
Indeed I think inlining is nicer in general ? I didn't realize that it's the final comma that triggers `black` indentation. There are a lot of other occurrences of indented lists of variables that can be inlined.

Reply:
I add inlining for almost all occurences, let me know what you think.